### DISCUSSION TITLE
Lit Review Choices

### DISCUSSION THREAD

**Discussion by annarupert** [ecn310/course-project-accidentsteam#5] [General]
Reply with the sources you have selected, cite in APA format please!

**Comment by annarupert** [comment]
As of now I am working on the first link that Professor Singleton gave us.

**Comment by ttakita-su** [comment]
https://journals.sagepub.com/doi/10.1177/0019793920958417

i will pick this

**Comment by ttakita-su** [comment]
What affect dose labor unions have on workplace safety 

Labor unions are known to maintain safe workplaces and prevent hazards in worksite though collective voice and communication with management. Dose data on the amount of workplace accidents from union and non-union Workers support this idea? 

 

Li, L., Rohlin, S., & Singleton, P. (2020). The Effect of Labor Unions on Workplace Safety: A National Study. Industrial Relations: A Journal of Economy and Society, 59(4), 643-670. 

 

 

How I found this paper. I found it online in the reference's professor Singleton used in his paper 

Big picture question: why dose the data show that there is no effect on unionization to accident case rates even though labor unions are known to make the workplace safer.  

Hypothesis – the people not in labor unions might not be educated on how to report an accident in the workplace. And it is easier to report something if a worker is in a union because the worker will have a bigger voice and support.  

Theory people in unions can report more accidents without backlash

**Comment by Yuhan-2435** [comment]
I will work on the second link and this article: https://doi.org/10.1177/0019793916687719. 
Riddell, W. C., & Song, X. (2017). The Role of Education in Technology Use and Adoption: Evidence from the Canadian Workplace and Employee Survey. ILR Review, 70(5), 1219–1253. https://doi.org/10.1177/0019793916687719

**Comment by annarupert** [comment]
1. Kniesner, T. J., & Leeth, J. D. (2014). Regulating occupational and product risks. In Handbook of the Economics of Risk and Uncertainty (Vol. 1, pp. 493-600). North-Holland.

**How I found this paper:** 
-  I used Professor Singletons research and found this as one of his sources. This seemed applicable to our out review assignment. 

**Big Picture Question:**  
- How do firms, households, and markets come together to determine the safety or health risk of a given workplace or good? 

**Hypothesis:** 
- There is evidence that households and firms come together to decide the safety of a job and the equivalent price associated with a job. Consumers and Workers equal compensation for this risk they are taking by working at a given workplace or using a good. 

**Theories that supported this hypothesis:**
-  Wages being dependent on the characteristics of the job. 
- Market prices for goods being dependent on their impact on consumers.
- Labor Markets (Law of Supply and Demand)


**Specific Question:** 
- How do labor and product markets pair prices and health risks and how does firm and government policy change post injuries or death?  Is there a way to increase safety without damaging profit? 

**Hypothesis:** 
- There is a substantial difficulty for workplaces to make health and safety improvement because of the finial impact it will have on the product price and the worker’s wages. However, there is a possibility that creating a safer workplace could allow wages to drop and increase profitability. 

**Theories that supported this hypothesis:** 
- Law of Supply and demand pertaining in this case to labor, if the price of one worker increases then the amount of workers demanded by a firm will decrease. 
- Government intervention in wages and workplace regulation.

**Comment by annarupert** [comment]
2. Levine, D. I., Toffel, M. W., & Johnson, M. S. (2012). Randomized government safety inspections reduce worker injuries with no detectable job loss. Science, 336(6083), 907-911.

**How I found this paper:** 
-  I used Professor Singletons research and found this as one of his sources. This seemed applicable to our out review assignment. 

**Big Picture Question:** 
- How do random workplace inspections impact the frequency of workplace accidents? 

**Hypothesis:** 
- When there are random inspections of firms safety and health risks, there are less workplace accidents. 

**Theories that supported this hypothesis:** 
- The principle of firm competition and how it interacts with inspections. 
- Cost of creating a safe workplace and how it impacts profit. 

**Specific Question:** 
- How do random occupational and health inspections impact the safety of workers and the frequency of workplace accidents. In the long run, how can this help lower injury-related costs for firms? 

**Hypothesis:** 
- When there are random inspections of firms safety and health risks, there are less workplace accidents, which in turn allows for less injury related costs to firms, including lawsuits. 

**Theories that supported this hypothesis:** 
- Cost to Firms 
- Potentially, an increase in profit or wages long term.,

**Comment by wmwaghor** [comment]
I am analyzing the following source,

Li, L., & Singleton, P. (2019). The Effect of Workplace Inspections on Worker Safety. _ILR Review, 72_(3), 718–748. https://doi.org/10.1177/0019793918801575

**Comment by wmwaghor** [comment]
What effects do workplace inspections have on workplace safety?

**How I found this paper:**
- This paper was provided as a resource by Professor Singleton, who also contributed to the paper, and it appears to be related to our assignment
**Big Picture Question:**
- How do workplace inspections impact the incidence rates of workplace injuries and accidents?
**Hypothesis:**
- Workplace inspections decrease the incidence rates of workplace injuries and accidents
**Theories Supporting this Hypothesis:**
- Punitive costs of violations vs costs/benefits of liability
**Specific Question:**
- Do site specific targeting (SST) plans impact workplace accidents, and do the benefits of this outweigh the marginal cost of implementing them
**Hypothesis:**
- SST plans do decrease the incidence of workplace accidents and their economic benefits outweigh the marginal costs of implementation
**Theories Supporting this Hypothesis:**
- Continuity of programmed inspections and DART rate
- Modeling SST policy on pre-implementation datapoints

**Comment by Yuhan-2435** [comment]
The article I found previously is somewhat digress from our project topic. I will probably change to another article. I will attach the link here later.

**Comment by Yuhan-2435** [comment]
I pick this article instead of the education one I chose previously. 
Demand Conditions and Worker Safety: Evidence from Price Shocks in Mining
https://www.journals.uchicago.edu/doi/10.1086/713887

**Comment by annarupert** [comment]
@Yuhan-2435 @wmwaghor @Tomoyoshikun Can you guys please update the Lit Review and reply to all of @kbuzard comments, you will find them with in your inbox. This is a group grade and we have a 3/5 on it as of right now, so please complete as soon as possible. If you cannot access them please email professor Buzard or text me and I can help as well. Thank you!!
